src/Extension/Task/DeprecationsCheckTask.yaml

```diff
@@ -4,7 +4,7 @@ short_description: "Check your code by running a static code analysis and genera
 help: ~
 stage: build
 version: 1.0.0
-command: "php %sdk_dir%/vendor/bin/psalm --config=%config% --no-progress --output-format=json --report=%reportDir%/psalm.deprecations_report.json 1> /dev/null"
+command: "php %sdk_dir%/vendor/bin/psalm.phar --config=%config% --no-progress --output-format=json --report=%reportDir%/psalm.deprecations_report.json 1> /dev/null"
```

Comment:
why `*.phar` ? There is no `psalm.phar` in `vendor/bin`

Reply:
Because changed to another package to be compatible with spryker projects

Reply:
Is it tested with all existing Spryker projects: b2b, b2c shops, and suite-nonsplit?